docs/examples/multi_modal/gemini.ipynb

```diff
@@ -0,0 +1,694 @@
+{
```

Comment:
<p>Better Title: "Build Multi-Modal RAG for Restaurant Recommendation". Sub-text can be "Our stack consists of Gemini + LlamaIndex + Pydantic structured output capabilities" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9452/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>